src/platform/PICO/mk/PICO.mk

```diff
@@ -25,20 +39,19 @@ DEVICE_STDPERIPH_SRC := \
             $(USBHID_SRC) \
             $(USBMSC_SRC)
 
+ifeq ($(TARGET_MCU),RP2350B)
```

Comment:
Is is necessary for standard PICO libraries?

Reply:
Probably not... I'm assuming different models may arrive

Reply:
Sorry, the comment was about LOWER and UPPER version of directory